styles.css

```diff
@@ -476,5 +488,240 @@ input:focus {
 .home-indicator {
   width: 134px;
   border-radius: 10px;
-  margin-top: 100px;
+  margin-top: 30px;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  html {
+    box-sizing: border-box;
+  }
+
+  .container {
+    width: auto;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .head {
+    width: 100%;
+    background-color: #fff;
+    position: fixed;
+  }
+
+  .navbar {
+    height: 72px;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .menu-items {
+    display: flex;
+    justify-content: space-around;
+    width: 250px;
+  }
+
+  /* headline section */
+  .headline {
+    height: 100vh;
+    background-image: url(./src/assets/img/desktop-head-back.svg);
+    background-size: contain;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .head-wraper {
+    margin-top: -120px;
+  }
+
+  .head-primary {
+    width: 920px;
+    height: 122px;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .head-intro {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    align-items: center;
+  }
+
+  /* Projects section */
+  .projects {
+    padding: 142px;
+    gap: 142px;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .project-card {
+    width: auto;
+
+    /* height: 496px; */
+    flex-direction: row;
+    align-items: center;
+    box-sizing: border-box;
+    padding-bottom: 16px;
+  }
+
+  .snapshoot img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .skills img {
+    width: 8px;
+    height: 14px;
+  }
+
+  .language img {
+    width: 48px;
+    height: 48px;
+  }
+
+  .snapshoot img:hover {
+    transform: scale(1);
+  }
+
+  .left-block {
+    padding: 24px;
+    width: 544px;
+  }
+
+  .second-card,
+  .fourth-card {
+    flex-direction: row-reverse;
+  }
+
+  .second-card br,
+  .fourth-card br {
+    display: none;
+  }
+
+  .project-title {
+    padding: 0;
+    width: 496px;
+    height: 98px;
+  }
+
+  .title {
+    height: 46px;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .frame ul {
+    width: 320px;
+    height: 40px;
+  }
+
+  .info {
+    width: 496px;
+    height: 70px;
+    font-size: 16px;
+  }
+
+  .project-tags ul {
+    width: 316px;
+    height: 24px;
+  }
+
+  /* About me section */
+  .about {
+    flex-direction: row;
+    padding: 144px;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: space-around;
+  }
+
+  .top {
+    width: 447px;
+  }
+
+  .skills {
+    width: 684px;
+  }
```

Comment:
- [ ] The same comment for the about and contact me sections, please don't fix the width, and please go through your styling and adjust any fixed width you set for the containers as it is holding your design from being flexible, also kindly correct the design so the content is showing up correctly (some part of the content is hidden on the left side), this is how it looks right now: 😁

![image](https://user-images.githubusercontent.com/20567503/179373954-bf2ea529-bf94-48dd-a1cd-35a5fc4af94a.png)